test/performance/odp_l2fwd.c

```diff
@@ -852,6 +1115,13 @@ static int create_pktio(const char *dev, int idx, int num_rx, int num_tx,
 	pktout_param.op_mode    = mode_tx;
 	pktout_param.num_queues = num_tx;
 
+	if (gbl_args->appl.vector_mode && pktio_capa.vector.supported) {
+		if (set_pktin_vector_params(&pktin_param, vec_pool, pktio_capa))
+			return -1;
+	} else {
+		printf("\nWarning: Vector mode parameters not set as feature is unsupported\n\n");
```

Comment:
You can just return failure here if vector mode is not supported.

Reply:
Agree.